docs/releases/3.1.txt

```diff
@@ -603,6 +603,11 @@ Miscellaneous
 * The admin CSS classes ``row1`` and ``row2`` are removed in favor of
   ``:nth-child(odd)`` and ``:nth-child(even)`` pseudo-classes.
 
+* The :func:`~django.contrib.auth.hashers.make_password` now requires its
+  argument to be a string or bytes to prevent accidental creation of
+  potentially weak passwords. Other types should be explicitly cast to one of
+  these.
```

Comment:
I'm a bit 😬 about the `to prevent accidental creation of
  potentially weak passwords`.

Maybe just: 

```suggestion
* The :func:`~django.contrib.auth.hashers.make_password` now requires its
  argument to be a string or bytes. Other types should be explicitly cast to
  one of these.
```

(Just a thought.)

Reply:
I tried to add a short rationale, but I'm fine to remove it.